### PR TITLE
My First Capstone Project

### PR DESCRIPTION
**Pull request by DJ-MrJay**
# In this project:

- I created the required _home_ and _about_ pages for both mobile and desktop following the design guidelines as close as possible
- Created HTML for a dynamic page section using JavaScript
- Implemented the hamburger menu toggle complete with menu items and a close icon
- Implemented a desktop navbar with menu items and a logo that links to the _home_ page
- Please [watch](https://www.loom.com/share/3f8211933628471db7931f84082676fb) the video presentation for more information.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -243,10 +250,13 @@ <h3>Partners</h3>
...
+      <br>
+      <span class="footer-additional">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nam pariatur vero sed fugiat soluta fuga rerum delectus similique excepturi magnam, odio dolorem, veritatis, numquam quod aut tempore maiores! Tenetur, natus. Lorem ipsum dolor sit amet consectetur adipisicing elit. Nam pariatur vero sed fugiat soluta fuga rerum delectus similique excepturi magnam, odio dolorem, veritatis, numquam quod aut tempore maiores! Tenetur, natus.</span></h6>
     </div>
   </footer>
   <script src="./main.js"></script>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (VICTOR-CHUKWU) [inline index.html:208]: - [ ] Your `about`  should be on a different HTML file and not inside your `index.html file`, Kindly create a file called `about.html` and copy paste your about section into it, onclick of `about in nav link` should navigate to the about page

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mr-jay/)
+
+## Contributors
+Contributions and feature requests are welcome!
+
+## Show your support
+Give ⭐️⭐️⭐️⭐️⭐️ if you like this project!
+
+## Github Page
+This project is [live](https://dj-mrjay.github.io/Capstone-1/) on Github Pages.
```
Comment (VICTOR-CHUKWU) [inline README.md:27]: - [ ] Your live page looks different from your project when run locally, Kindly fix

Comment (VICTOR-CHUKWU) [inline README.md]: - [ ] Please add a video describing this project, this video should not be more than 5 mins

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @DJ-MrJay , 
## STATUS: CHANGES REQUIRED :recycle: 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
 # Capstone-1
 My first capstone project at Microverse
+
```
Comment (Deepakdanger) [inline README.md]: - [ ] I think, we should give some proper name and description about the project.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @DJ-MrJay , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] As requested by the previous reviewer, The student recorded a video presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a clear and objective description about the project in professional English. - The topic and purpose of the project are made clear.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.
- > **Note:** It's recommended that students use [Loom](https://www.loom.com/) to record to video but they can use any tool they want as long as they share an online link of the uploaded video.
- [ ] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (DJ-MrJay) [reply README.md]: Such as?

**Review by Kingstalux — APPROVED** [review]
Hi @DJ-MrJay,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :clap: 
:white_check_mark: Great video presentation :fire: 
:white_check_mark: Good UI/UX design :ok_hand: 
:white_check_mark: Good commit history :+1: 
:white_check_mark: Professional README file :100: 

- [ ] (OPTIONAL) On the mobile version of the home page, only 2 speakers should be displayed together with the `more` button. Then when a user clicks on the `more` button, the rest of the speakers will then be displayed. This functionality will improve the UX of your site :fire: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**